### PR TITLE
Fix CouchDB and Python tests

### PR DESCRIPTION
**Pull request by cesar-cs**
## What is the problem that this PR is trying to fix?
CouchDB and Python tests

## What approach did you choose and why?
Tests for CouchDB will verify the OS before choosing to make authenticated requests.
Fixed Python versions.

### CODE REVIEW

File: cookbooks/lib/features/couchdb_spec.rb [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # frozen_string_literal: true
 
+def couchdb_url
+  return 'http://127.0.0.1:5984' if %w[trusty xenial].include?(Support.distro)
+  return 'http://admin:travis@127.0.0.1:5984' if 'bionic'.include?(Support.distro)
+
+  'http://127.0.0.1:5984'
```
Comment (DamianSzymanski) [inline cookbooks/lib/features/couchdb_spec.rb:7]:
```suggestion
  'http://admin:travis@127.0.0.1:5984'
```

Comment (DamianSzymanski) [reply cookbooks/lib/features/couchdb_spec.rb:7]: I propose this as default return, we also have focal and probably more distros later

File: cookbooks/lib/languages/python_spec.rb [hunk 2]
```diff
@@ -39,10 +39,9 @@
   end
 
   {
-    'python2.7' => '2.7.17',
-    'python3.6' => '3.6.10',
-    'python3.7' => '3.7.6',
-    'python3.8' => '3.8.1'
+    'python2.7' => '2.7.15',
+    'python3.6' => '3.6.7',
+    'python3.7' => '3.7.1'
```
Comment (DamianSzymanski) [inline cookbooks/lib/languages/python_spec.rb:44]: What about other distros, i propose to add some conditions here if xenial, if bionic etc

Comment (cesar-cs) [reply cookbooks/lib/languages/python_spec.rb]: Done! ✅ 
Thanks for the suggestions!

**Pull request merged**